docs/process.rst

```diff
@@ -4,13 +4,14 @@
 Processes
 *********
 
-In Nextflow a `process` is the basic processing `primitive` to execute a user script.
+In Nextflow a *process* is the basic processing *primitive* to execute a user script.
 
-The process definition starts with the keyword ``process``, followed by process name and finally the process `body`
+The process definition starts with the keyword ``process``, followed by process name and finally the process *body*
 delimited by curly brackets. The process body must contain a string which represents the command or, more generally,
 a script that is executed by it. A basic process looks like the following example::
 
   process sayHello {
+      echo true
```

Comment:
Think here `echo` should not be added since the stdout is redirected to the file.